README.md

```diff
@@ -95,3 +95,11 @@ $ find . -type f -exec puppet-lint --only-checks ghostbuster_classes,ghostbuster
 ./modules/foo/files/bar.txt - WARNING: File foo/bar.txt seems unused on line 1
 ./modules/foo/files/baz.txt - WARNING: File foo/baz.txt seems unused on line 1
 ```
+
+## Transfer Notice
```

Comment:
please also add the label at the top

Reply:
fixed the flags, test and release should properly display once modulesync is done